waku/waku_rln_relay/group_manager/on_chain/group_manager.nim

```diff
@@ -234,19 +238,19 @@ when defined(rln_v2):
     let membershipFee = g.membershipFee.get()
 
     var gasPrice: int
-    retryWrapper(gasPrice, RetryStrategy.new(), "Failed to get gas price"):
+    g.retryWrapper(gasPrice, RetryStrategy.new(), "Failed to get gas price"):
```

Comment:
Shouldn't we delete here the `RetryStrategy.new()`?
Same in some other places in this file

Reply:
slipped thru the cracks, apologies - 8d27b961

Reply:
interesting that it compiles anyway :O